hooks/check-projects

```diff
@@ -9,7 +9,7 @@ if ! [[ -f "$SOLUTION" ]]; then
 fi
 
 list-projects() {
-  dotnet sln "$SOLUTION" list | tail -n +3 | awk -F/ '{ print $1 }'
+  dotnet sln "$SOLUTION" list | grep '\...proj$' | awk -F/ '{ print $1 }'
```

Comment:
Is it for what files? It can include other files, not csproj.

Reply:
What do you mean?  `dotnet sln "$SOLUTION" list` prints the list of MSBuild project files and some header lines too. Previously I had assumed the header take only 3 lines, but it apparently depends on .NET Core SDK's version and environment.  So I've changed the heuristics so that only lines that ends with `\...proj` (i.e., `.*proj`; e.g., `.csproj`, `.vbproj`).

Reply:
I forgot briefly there can be other extensions matched to `*proj` (e.g., `.vbproj`, `.fsproj`). I understood. 🙏